### PR TITLE
Calculate mean, median and standard deviation sentiment from Reddit comments and load into database

### PR DESCRIPTION
**Pull request by rk145lr**
- Contains the changes for the implementation of the transform and load stages for the Reddit pipeline.
- The mean, median and standard deviation of the sentiment of the comments for a page is calculated (uses the same calculation approach as the RSS pipeline).
- A single row is added to the database for each page in the subreddit. This includes the title, domain, url, number of comments (on the page and processed), upvote ratio, timestamp of creation. If a subreddit url is already present in the database the row is updated with the latest values.
- Unit tests have been created which test the core functionality as well as edge cases.
- `The README.md` has been updated with the required environment variables.

### CODE REVIEW

File: public_sentiment_pipeline/load.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+def establish_database_connection(config: dict):  # pragma: no cover
+    """Establishes a connection with the PostgreSQL RDS database."""
+    try:
+        return connect(
+            user=config['DATABASE_USERNAME'],
+            password=config['DATABASE_PASSWORD'],
+            host=config['DATABASE_ENDPOINT'],
+            port=config['DATABASE_PORT'],
+            database=config['DATABASE_NAME'])
```
Comment (ramiz76) [inline public_sentiment_pipeline/load.py:34]: I've been using environ to retrieve my .env values so we may need to come together and decide what styles/formatting we'd all like to use to keep it consistent

**Review by ramiz76 — APPROVED** [review]

**Review by isaac-vb — APPROVED** [review]
